### PR TITLE
Basic call transfer initiation support

### PR DESCRIPTION
**Pull request by dbkr**
New PR for https://github.com/matrix-org/matrix-js-sdk/pull/1558 after accidentally merging 1559 which was based on it.

### CODE REVIEW

**Comment by dbkr** [comment]
Original PR feedback incorporated - should just need a quick re-review

File: src/webrtc/call.ts [hunk 1]
```diff
@@ -779,7 +810,14 @@ export class MatrixCall extends EventEmitter {
                 // required to still be sent for backwards compat
                 type: this.peerConn.localDescription.type,
             },
-        };
+        } as any;
```
Comment (jryans) [inline src/webrtc/call.ts:813]: Should this be typed as well?

File: src/webrtc/call.ts [hunk 2]
```diff
@@ -1102,7 +1141,13 @@ export class MatrixCall extends EventEmitter {
         const content = {
             [keyName]: this.peerConn.localDescription,
             lifetime: CALL_TIMEOUT_MS,
-        };
+        } as any;
```
Comment (jryans) [inline src/webrtc/call.ts:1144]: Should this be typed as well?

**Review by jryans — APPROVED** [review]

File: src/webrtc/call.ts (hunk 1 above)
Comment (dbkr) [reply src/webrtc/call.ts]: ah yep - fixed

File: src/webrtc/call.ts (hunk 2 above)
Comment (dbkr) [reply src/webrtc/call.ts]: likewise

**Pull request merged**